GaugeKit.xcodeproj/project.pbxproj

```diff
@@ -330,6 +331,7 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				ALWAYS_SEARCH_USER_PATHS = NO;
+				CLANG_ANALYZER_LOCALIZABILITY_NONLOCALIZED = YES;
```

Comment:
new flag. I didn' saw it in my vhanges. what does it do?

Reply:
This fixes a localization warning. I'll revert it.